mm/Kconfig

```diff
@@ -194,6 +194,15 @@ config MM_SHM
 		Build in support for the shared memory interfaces shmget(), shmat(),
 		shmctl(), and shmdt().
 
+config MM_UMAP
```

Comment:
let's reuse MM_SHM config? the new functions doesn't have any side effect if nobody call them.